### PR TITLE
/etc/grc.conf : KubeCTL : remove a regular expression "^" too much

### PR DESCRIPTION
**Pull request by Raizo62**
Hi

Could you see also the pull request by command here : #130  ?

### CODE REVIEW

File: grc.conf [hunk 1]
```diff
@@ -195,7 +195,7 @@ conf.dockerversion
 conf.log
 
 # KubeCTL
-^(^|[/\w\.]+/)kubectl((?!edit|exec|run|go-template).)*$
+(^|[/\w\.]+/)kubectl((?!edit|exec|run|go-template).)*$
```
Comment (luisdavim) [inline grc.conf:198]: wouldn't

```
^([/\w\.]+\/)?kubectl((?!edit|exec|run|go-template).)*$
```
be a better option?

Comment (Raizo62) [reply grc.conf:198]: I believe that you are right.
But your change is not only for kubectl but for all commands.

It needs to write a sed command to change all the commands on the last file grc.conf

Comment (Raizo62) [reply grc.conf:198]: I don't understand why you don't merge this : it is clear that the regular expression of kubectl in /etc/grc.conf is not the same than the others

**Comment by Raizo62** [comment]
I don't understand why you don't merge this : it is clear that the regular expression of kubectl in /etc/grc.conf is not the same than the others

**Comment by garabik** [comment]
> I don't understand why you don't merge this : it is clear that the regular expression of kubectl in /etc/grc.conf is not the same than the others

Just an oversight - I am fixing all the other entries as well with 826422fad6f8136b8b4e57c9d60bc2469d2a4941

**Comment by garabik** [comment]
Fixed in 826422fad6f8136b8b4e57c9d60bc2469d2a4941

**Pull request closed without merging**